app/views/shared/_profile_card_content.html.erb

```diff
@@ -43,7 +43,7 @@
         <% header_fields.sort.each do |title, value| %>
           <li>
             <div class="key">
-              <%= t("views.users.profile_fields.#{title}") %>
+              <%= t("views.users.profile_fields.#{title.downcase}", default: title.titleize) %>
```

Comment:
I'm downcasing here, because title comes from ui_attributes_for(area), which is the label (and likely titleized, especially in the case of Work and Education from the seeds). The translation keys are work: and education: and it feels faster to adjust before translating.

I'm titleizing the display in the default case if it wasn't (so the "feel" of the profile cards stays consistent).